server/Arcadia.Assistant.Calendar/SickLeave/EmployeeSickLeaveActor.cs

```diff
@@ -10,17 +10,19 @@
 
     public class EmployeeSickLeaveActor : CalendarEventsStorageBase
     {
-        public EmployeeSickLeaveActor(string employeeId)
+        public EmployeeSickLeaveActor(string employeeId, IActorRef sendEmail)
```

Comment:
-> sendEmailActor.

It is an object/actor, so it should usually be called with noun

Reply:
or, rather, sickListNotifications. Here we don't want to know if it sends an email, or telegram message, or calls api, we just want to tell to notify people.